src/i_video.c

```diff
@@ -750,7 +750,9 @@ void I_FinishUpdate(void)
             remaining_time = target_time - elapsed_time;
 
             if (remaining_time > 1000)
-                I_Sleep((remaining_time - 1000) / 1000);
+            {
+                I_Sleep(1);
+            }
```

Comment:
The `I_Sleep(1);` change makes the issue occur more frequently.

I did a partial revert of this commit and it seems to have fixed the issue: https://github.com/fabiangreffrath/woof/commit/a8b898a8139d28127e3ecc79496f019b865d6d48

These are the changes. Only capped behavior needs to be reverted like so;

```diff
diff --git a/src/d_loop.c b/src/d_loop.c
index 82c157e6..d1844258 100644
--- a/src/d_loop.c
+++ b/src/d_loop.c
@@ -729,8 +729,7 @@ void TryRunTics (void)
 
     // [AM] If we've uncapped the framerate and there are no tics
     //      to run, return early instead of waiting around.
-    // Only check counts so that I_FinishUpdate() can handle frame timing.
-    #define return_early (counts == 0)
+    #define return_early (uncapped && counts == 0)
 
     // get real tics
     entertic = I_GetTime() / ticdup;
diff --git a/src/i_video.c b/src/i_video.c
index 8c609d9b..270cc038 100644
--- a/src/i_video.c
+++ b/src/i_video.c
@@ -372,7 +372,7 @@ static void UpdateLimiter(void)
     }
     else
     {
-        use_limiter = (targetrefresh > 0);
+        use_limiter = false;
     }
 }
```